### ISSUE TITLE
Convert private variables into public

### ISSUE THREAD

**Issue by gbalabasquer** [MakerDAO/token-auction#60]

**Comment by gbalabasquer** [comment]
_last_auction_id can be got from the LogNewAuction Event, but we do not have the same situation with auctionlets (_last_auction_id).
I would still suggest to make both (_last_auctionlet_id and _last_auction_id) public if possible.

**Comment by rainbreak** [comment]
I think there is a solution to this as it stands though:
- when a new auction is created, LogNewAuction is emitted with the auction id _and_ the base auctionlet id
- any bids on this auctionlet id can be seen through the `LogBid(auctionlet_id)` event

With no splitting, the auctionlet_id doesn't change and you can use the value from `LogNewAuction` as the last created auctionlet.

I'm still not sure exactly what you're wanting to to with `_last_auction(let)_id`. Is it:
1. Use that number as the end of a loop for iterating through the auction(let)s, getting the info for each one?
2. Use that number to know when to stop consuming the event stream?

In the case of (1) I'd say that you can't rely on the auction(let)_id simply incrementing. (2) is ok I think.

**Comment by gbalabasquer** [comment]
Yes with no splitting we can actually use the base auctionlet_id or directly the auction_id as they are the same.
When splitting is implemented we could use both events mixed as you said. It will be a bit more complicated in the frontend to get the number but still should work.

What I do not understand is why we could not rely in _last_auction(let)_id to know the maximum auctionlet_id and iterate until reach that number.
